.github/workflows/tests.yml

```diff
@@ -55,12 +55,19 @@ jobs:
         with:
           username: locustbuild
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      - uses: docker/build-push-action@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: build and publish image
+        id: docker_build
+        uses: docker/build-push-action@v2
+        if: ${{ github.ref == 'refs/heads/master' && github.repository_owner == 'locustio' }}
         with:
-          build-args: |
-            BUILDKIT_CONTEXT_KEEP_GIT_DIR=1
```

Comment:
The reason I had to set this flag was so that the .git folder is available for docker (without it, it setuptools_scm cannot detect the version and will say something stupid like 0.0.1dev1234 instead of 2.2.4dev123)